CHANGELOG.md

```diff
@@ -7,6 +7,98 @@ documentation before upgrading to a new release.
 
 Released closed milestones can be found on [GitHub](https://github.com/Icinga/icinga2/milestones?state=closed).
 
+## 2.13.3 (2022-04-14)
+
+This version includes bugfixes for many features of Icinga 2, including fixes for multiple crashes. It also includes a
+number of fixes and improvements for Icinga DB.
+
+### API
+
+* The /v1/config/stages endpoint now immediately rejects parallel config updates instead of accepting and then later
+  failing to verify and activate them. #9328
```

Comment:
Please rethink your strategy as in https://github.com/Icinga/icinga2/pull/8944#discussion_r681002942 .